scripts/infra/instructlab-cloud-config-example

```diff
@@ -11,6 +11,7 @@ EC2_SUBNET_ID="subnet-12345678901234567"
 EC2_SUBNET_VARS=("EC2_SUBNET_ID_1" "EC2_SUBNET_ID_2")
 EC2_SUBNET_ID_1=subnet-12345678901234567
 EC2_SUBNET_ID_2=subnet-12345678901234568
+# Omit EC2_AMI_ID to use the latest CentOS Stream 9 image.
```

Comment:
Should this be the default? (Comment out the particular AMI selection and instruct user that they can hardcode the image if needed?)

Reply:
Good idea. I've amended and repushed.